pkg/model/rhsso_deployment_test.go

```diff
@@ -31,3 +35,49 @@ func TestRHSSODeployment_testAffinityExperimental(t *testing.T) {
 func TestRHSSODeployment_testDeploymentSpecImagePolicy(t *testing.T) {
 	testDeploymentSpecImagePolicy(t, RHSSODeployment)
 }
+
+func TestRHSSODeploymentReconciled_testDisableRHSSOReplicasSyncingFalse(t *testing.T) {
+	testDisableRHSSOReplicasSyncingFalse(t, RHSSODeployment, RHSSODeploymentReconciled)
+}
+
+func TestRHSSODeploymentReconciled_testDisableRHSSOReplicasSyncingTrue(t *testing.T) {
+	testDisableRHSSOReplicasSyncingTrue(t, RHSSODeployment, RHSSODeploymentReconciled)
+}
+
+func testDisableRHSSOReplicasSyncingFalse(t *testing.T, deploymentFunction createDeploymentStatefulSet, deploymentFunction2 reconcileRHSSODeployment) {
```

Comment:
Just to align with other tests in this file, could you please move this to `keycloak_deployment_test`?

Reply:
Thank you @vmuzikar, done!